templates/workloads/partials/cf/servicediscovery.yml

```diff
@@ -16,4 +16,8 @@ DiscoveryService:
     Name:  !Ref WorkloadName
     NamespaceId:
       Fn::ImportValue:
+      {{- if .LegacyServiceDiscovery}}
         !Sub '${AppName}-${EnvName}-ServiceDiscoveryNamespaceID'
+      {{- else}}
+        !Sub '${AppName}-${EnvName}-ServiceDiscoveryNamespaceIDWithEnv'
+      {{- end}}
```

Comment:
Do we want to include both if they recently upgraded their environment? since in the env template if they were using legacy sd we also add new sd for them. Or we don't create new sd if they are using legacy at all.